### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by aaronpeikert**
@moritzketzer Could you please amend your information here?

### CODE REVIEW

**Comment by Maximilian-Stefan-Ernst** [comment]
@moritzketzer please amend your information.

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - family-names: Ernst
+    given-names: Maximilian Stefan
+    email: ernst@mpib-berlin.mpg.de
+    affiliation: Max Planck Institute for Human Development
+    orcid: 'https://orcid.org/0000-0003-2237-6255'
+  - family-names: Ketzer
+    given-names: Moritz
+repository-code: >-
+  https://github.com/formal-methods-mpi/Workshop.jl/
+license: CC0
```
Comment (aaronpeikert) [inline CITATION.cff:21]: Require #117

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    email: peikert@mpib-berlin.mpg.de
+    affiliation: Max Planck Institute for Human Development
+    orcid: 'https://orcid.org/0000-0001-7813-818X'
+  - family-names: Ernst
+    given-names: Maximilian Stefan
+    email: ernst@mpib-berlin.mpg.de
+    affiliation: Max Planck Institute for Human Development
+    orcid: 'https://orcid.org/0000-0003-2237-6255'
+  - family-names: Ketzer
+    given-names: Moritz
```
Comment (moritzketzer) [inline CITATION.cff:18]:
```suggestion
    given-names: Moritz
    email: moritz.ketzer@hu-berlin.de
    affiliation: Max Planck Institute for Human Development; Humboldt Universität zu Berlin
    orcid: 'https://orcid.org/0000-0002-2403-5297'
```

Comment (aaronpeikert) [reply CITATION.cff:18]: I am pretty sure that multiple affiliations are handled differently, could you check?

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+authors:
+  - given-names: Aaron
+    family-names: Peikert
+    email: peikert@mpib-berlin.mpg.de
+    affiliation: Max Planck Institute for Human Development
+    orcid: 'https://orcid.org/0000-0001-7813-818X'
+  - family-names: Ernst
+    given-names: Maximilian Stefan
+    email: ernst@mpib-berlin.mpg.de
+    affiliation: Max Planck Institute for Human Development
```
Comment (Maximilian-Stefan-Ernst) [inline CITATION.cff:15]:
```suggestion
    affiliation: Max Planck Institute for Human Development & Humboldt Universität zu Berlin
```

Comment (Maximilian-Stefan-Ernst) [inline CITATION.cff:15]:
```suggestion
    affiliation: "Max Planck Institute for Human Development & Max Planck School of Cognition
```

File: CITATION.cff (hunk 2 above)
Comment (moritzketzer) [reply CITATION.cff:18]: I prefer the semicolon on one line.
We could also use two lines?

```suggestion
    affiliation: Max Planck Institute for Human Development
    affiliation: Humboldt Universität zu Berlin
```

**Comment by moritzketzer** [comment]
> I prefer the semicolon on one line.

Because I feel that an ampersand might imply they're part of a single entity.

```

**Comment by moritzketzer** [comment]
> I prefer the semicolon on one line.

Because I feel that an ampersand might imply they're part of a single entity.
```

**Comment by moritzketzer** [comment]
I just stumbled upon [this](https://quarto.org/docs/authoring/front-matter.html) in quarto. could we maybe nest the affiliation like the next example?

---
author:
  - name: Norah Jones
    affiliation: 
      - Carnegie Mellon University
      - University of Chicago
  - name: Josiah Carberry
    affiliation: Brown University
---

**Pull request merged**